store/store-core/src/main/java/org/eclipse/lyo/store/internals/SparqlStoreImpl.java

```diff
@@ -286,17 +298,17 @@ public Model getResources(final URI namedGraph, final String prefixes, final Str
             }
     	}
 
-    	SelectBuilder sparqlWhereQuery = constructSparqlWhere (prefixes, where, searchTerms, limit, offset);
+    	SelectBuilder sparqlWhereQuery = constructSparqlWhere (prefixes, where, searchTerms, limit, offset, additionalDistinctVars, additionalQueryFilter);
     	DescribeBuilder describeBuilder = new DescribeBuilder();
-		if (namedGraph != null) {
-			describeBuilder.addVar("s")
-			.addGraph(new ResourceImpl(String.valueOf(namedGraph)), sparqlWhereQuery);
-		}
-		else {
-			describeBuilder.addVar("s")
-			.addGraph("?g", sparqlWhereQuery);
-		}
-
+		describeBuilder.addVar("s")
+		.addGraph((namedGraph != null) ? new ResourceImpl(String.valueOf(namedGraph)) : "?g", sparqlWhereQuery);
+		
+        if (null != additionalDistinctVars) {
+        	for (String var : additionalDistinctVars) {
+        		describeBuilder.addVar(var);
```

Comment:
is it possible to fix mix of tabs and spaces on the same line, please?

Reply:
Yes. done.